steadybit/README.md

```diff
@@ -0,0 +1,66 @@
+# Steadybit
+
+## Overview
+
+[Steadybit][1] is a chaos engineering platform allowing you to simulate turbulent conditions in a controlled way, to improve system reliability and guide your organization to better incident management.
+
+Through this integration, you can leverage Datadog information within chaos engineering experiments and see chaos engineering activity in the form of events within Datadog.
```

Comment:
Thank you for making the previous changes! Could you provide insight on:
1. What type of information customers can leverage? 
2. What type of information is sent to datadog via events?

We typically like the overview clear enough so someone who isn't familiar with the integration will be able to read the overview and understand what the partner does, what the integration does and the value it provides :)